.github/workflows/e2e.yml

```diff
@@ -106,6 +102,27 @@ jobs:
         with:
           CWD: "../generated-projects/retail-app-no-ext"
           TARGET: e2e-tests-pwa-kit
+
+      - name: Wait for MRT Deploy to complete
+        uses: "./.github/actions/wait_for_mrt_deploy"
+        with:
+          TARGET: e2e-tests-pwa-kit
+          MRT_API_KEY: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
+          VERSION: ${{ github.sha }}
+          PROJECT: scaffold-pwa
+
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+
+      - name: Run Playwright tests
+        run: npx playwright test
+
+      - uses: actions/upload-artifact@v3
+        if: always()
```

Comment:
What does this do?

Reply:
Running playwright tests generates an html file with the report of test results which need to be uploaded to github actions storage as artifacts to be accessed later. The `actions/upload-artifact@v3` internally handles skipping the step if no report file exists in-case of the tests failing to start at all. So `always()` will run this step irrespective of whether the tests step fails or passes.